service/handlers.go

```diff
@@ -271,6 +273,19 @@ func handleCommit(c *Core, w *ResponseWriter, r *Request) {
 	if err != nil {
 		w.Error(err)
 	}
+
+	b, err := json.Marshal(struct {
+		CommitID string `json:"commit_id"`
+		PoolID   string `json:"pool_id"`
+	}{
+		CommitID: commitID.String(),
+		PoolID:   pool.ID.String(),
+	})
```

Comment:
```suggestion
	b, err := json.Marshal(struct {
		CommitID ksuid.KSUID `json:"commit_id"`
		PoolID   ksuid.KSUID `json:"pool_id"`
	}{
		CommitID: commitID,
		PoolID:   pool.ID,
	})
```

Reply:
We keep all the api responses in the api package. I would create a struct there called `EventPoolCommit`.